pkg/models/audit_change.go

```diff
@@ -77,3 +80,14 @@ func (a *AuditFields) Scan(src interface{}) error {
 
 	return nil
 }
+
+// ModifiedByUserAccount returns the user account of the user who created the struct from the DB using the UserAccount service
+func (ac *AuditChange) ModifiedByUserAccount(ctx context.Context) *authentication.UserAccount { //TODO should this be moved to a shared struct? This isn't a base struct
```

Comment:
I think it is, I didn't want to try to implement something with this in this PR to avoid scope creep. But I do think we could further extend the embedded struct philosophy if we want. Most objects have a modifiedBy field, even if they don't completely implement I-BaseStruct. We could potentially further extend that, and make the fields themself be their own embedded struct, so base struct embeds a modified_by struct etc. I'm just not sure if it's the best choice yet, but I want to leave a note.